src/molecule/command/test.py

```diff
@@ -55,6 +55,13 @@ class Test(base.Base):
 
         Target all scenarios.
 
+    .. option:: molecule test -- -vvv --tags foo,bar
+
+        Providing additional command line arguments to the `ansible-playbook`
+        command.  Use this option with care, as there is no sanitation or
+        validation of input.  Options passed on the CLI override options
+        provided in provisioner's `options` section of `molecule.yml`.
```

Comment:
This is not true since those options are combined: https://github.com/ansible-community/molecule/blob/60b68140fb5c650c47019f5db238c0864dbd43ed/src/molecule/provisioner/ansible_playbook.py#L75-L76

Reply:
I just made a copy of this description from here:
https://github.com/ansible-community/molecule/blob/60b68140fb5c650c47019f5db238c0864dbd43ed/src/molecule/command/converge.py#L53-L54

Reply:
That description is also wrong and should be fixed, but not in this PR since it has nothing to do with adding a new option to the test subcommand.